src/string.rs

```diff
@@ -41,10 +35,7 @@ fn find_char_midpoint(chars: &str) -> usize {
 
 /// Parallel extensions for strings.
 pub trait ParallelString {
-    /// This trait is private; this method exists to make it impossible
-    /// to implement outside the crate.
-    #[doc(hidden)]
-    fn private_parallel_string(&self) -> private::PrivateMarker;
+    private_decl!{}
```

Comment:
Done.

I was hoping to sneak that in with a `//!` in `private_decl!`, but that doesn't actually work.  Alas.